content_scripts/doctolib/book.js

```diff
@@ -36,6 +36,25 @@
     await wait();
   }
 
+  // Parfois on doit envoyer un vrai click avec tous les événements.
+  function fireFullClick(target) {
+    ["mousedown", "mouseup", "click"].forEach(type => {
+      const evt = new MouseEvent(type, { bubbles: true, cancelable: true });
+      target.dispatchEvent(evt);
+    });
+  }
+
+  function isARNmMotive(text) {
+    // Matches "Vaccin Pfizer" but not "2de dose Pfizer suite à 1e dose AstraZeneca"
+    return (text.includes("Pfizer") || text.includes("Moderna")) && !text.startsWith("2");
+  }
+
+  function isGeneralPopulationMotive(text) {
+    // Matches "Patients de 18 à 50 ans" but not "Patients de plus de 50 ans"
+    // Matches "Je suis un particulier"
+    return text.includes("18") || text.includes("particulier");
```

Comment:
Il me semble avoir vu passer des "plus de 18 ans avec comorbidité", j'ai peur que ce test ne soit pas assez restrictif.

Reply:
Aaaah c'est possible... Je vais mettre "18 à 50" dans ce cas ?

Reply:
Pas sûr que ça règle le problème en fait. Peut-être :
```suggestion
    return (text.includes("18") || text.includes("particulier")) && !text.includes("comorb");
```

Je ne me souviens plus du texte exact malheureusement.

Reply:
ça me va